README.md

```diff
@@ -1,126 +1,190 @@
 <a name="readme-top"></a>
 
-<div align="center">
 
-  <br/>
+<!-- TABLE OF CONTENTS -->
 
-  <h1><b>Mobile Portfolio</b></h1>
-
-</div>
-
-## 📗 Table of Contents
+# 📗 Table of Contents
 
 - [📖 About the Project](#about-project)
-  - [🛠 Built With](#built-with)
-    - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
+  - [🛠️ Built With](#built-with)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
   - [Run tests](#run-tests)
-  - [Deployment](#deployment)
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
+- [⭐ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
 - [📝 License](#license)
```

Comment:
- [ ] Please🙏🙏 ensure that you include the appropriate README template with all the necessary headers and their corresponding descriptions in your repository. 
- [ ] This step is crucial to maintain a professional appearance for your project, and it is also a project requirement.
- [ ]  A well-structured and informative README provides important details about your project, its purpose, installation instructions, usage guidelines, and other relevant information that will benefit users and potential contributors.🤔🤔🤔

Reply:
Hello, thank you for your review. Please I'm a little confused here. The README file you are seeing was readjusted by the previous code reviewer. This was done through a zoom call. I'm getting confused on which one to adopt finally.

Reply:
Thanks, @ngalamac for your concern, Problem is you change the entire readme file from your previous code base.